draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -2682,18 +2682,19 @@ https://video.example.com/k/movie1/4/013.ts
 
         </list></t>
 
-        <t>Object specification<list style="empty">
-
-              <t>Property: locations<list style="empty">
-
-                 <t>Description: An Access List that allows or denies (blocks) the trigger execution
-                    per cache location.</t>
+	<t>Object specification:</t>
+        <t><list style="empty" hangIndent="3">
+        <t><list style="hanging" hangIndent="3"><t hangText="Name: locations">
+        <vspace blankLines="1"/>
+	Description: An Access List that allows or denies (blocks) the trigger execution per cache location.
+        <vspace blankLines="1"/>
+        Value: Array of LocationRule objects (see Section 4.2.2.1 of <xref target="RFC8006"/>).
```

Comment:
Type vs. Value: So this was "Type" before, and you converted it to "Value".  It appears that "Type" and "Value" are used fairly interchangeably throughout the document (with Type appearing a little more often than Value).  Should we look at making this more consistent throughout the document?

Reply:
I think these are multiple editing layers / editors
Propose to standardize on "Name" and "Value"

Reply:
Marking as resolved as well